electrician/demo/sale_order_line.xml

```diff
@@ -0,0 +1,134 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<odoo noupdate="1" auto_sequence="1">
+  <record id="sale_order_line_15" model="sale.order.line">
+    <field name="order_id" ref="sale_order_10"/>
+    <field name="product_id" ref="product_product_5"/>
+    <field name="analytic_distribution" eval="{'%s'%(ref('account_analytic_account_8')): 100.0}"/>
```

Comment:
This shouldn't be done here, but it is automatic if you add the group_analytic_accounting group in the settings.
```suggestion
```